From ee734ebbd220460e73cc871d1f0c652bb9a85439 Mon Sep 17 00:00:00 2001
From: Alexander Aring <alex.aring@googlemail.com>
Date: Fri, 21 Dec 2012 06:37:14 +0100
Subject: [PATCH 3/7] configure.ac: check for python2.x

Check for python2.x at first.

test_DQ.py works with python2.x only.
'make install' needs python2.x to generate *.pyc files.

Signed-off-by: Alexander Aring <alex.aring@gmail.com>
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index c483b4f..6c9f54b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -77,6 +77,9 @@ AC_PROG_LIBTOOL
 AM_PROG_LEX
 AC_PROG_YACC
 AC_SUBST(LIBTOOL_DEPS)
+# Currently the scripts works with python2.x only
+# Check for this at first
+AC_PATH_PROG(PYTHON, python2 python2.7 python2.6 python2.5 python)
 AM_PATH_PYTHON(,, [:])
 AM_MISSING_PROG(HELP2MAN, help2man, $missing_dir)
 
-- 
1.8.2.3

